packages/orchestration/src/exos/cosmos-orchestration-account.js

```diff
@@ -448,7 +488,102 @@ export const prepareCosmosOrchestrationAccountKit = (
             Fail`'amountToCoin' not working for ${q(amount.brand)} until #10449; use 'DenomAmount' for now`;
           return coerceCoin(chainHub, amount);
         },
+        /**
+         * Submit a transaction on behalf of the remote account for execution on
+         * the remote chain.
+         *
+         * Set `relativeTimeoutNs` to provide a timeout for the IBC packet.
+         *
+         * `TxBody` fields like `timeoutHeight` and `memo` can be set, but these
+         * typically do not affect IBC app protocols like PFM, ICA.
+         *
+         * @param {[
+         *   ...Parameters<HostOf<IcaAccount['executeEncodedTx']>>,
```

Comment:
I suggest writing out the param types explicitly, using a type test to ensure consistency.